src/library/Uno.Material/Styles/Controls/v1/TextBox.xaml

```diff
@@ -154,8 +154,6 @@
 									<Storyboard>
 										<ObjectAnimationUsingKeyFrames Storyboard.TargetName="DeleteButton"
 																	   Storyboard.TargetProperty="Opacity">
-											<DiscreteObjectKeyFrame KeyTime="0"
-																	Value="1" />
```

Comment:
Does this also need to be done in the v2 styles for TextBox?

Reply:
We don't even know why it was added in the first place. We could also do it in V2 but I don't know if it's worth it.

Reply:
Like Thomas said someone added this delay in v1 for an unknown reason. In my opinion the delay animation shouldn't be there at all since there is no delay in the winui gallery app for textbox.